src/webui/www/private/shareratio.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+<head>
+    <meta charset="UTF-8" />
+    <title>QBT_TR(Torrent Upload/Download Ratio Limiting)QBT_TR[CONTEXT=UpDownRatioDlg]</title>
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <script src="scripts/mootools-1.2-core-yc.js"></script>
+    <script src="scripts/mootools-1.2-more.js"></script>
```

Comment:
Since #8672 is merged, please rebase this PR and also you'll need to change this to: `scripts/lib/mootools-1.2-core-yc.js`, `scripts/lib/mootools-1.2-more.js`.